pyapns_client/notification.py

```diff
@@ -164,22 +165,34 @@ def __init__(
             # Recommended to use dictionary, see https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/generating_a_remote_notification#2943360
             self.alert = _PayloadAlert(body=alert)
         else:
-            raise ValueError("alert must be a string or a _PayloadAlert object")
+            value_type = type(self.alert)
```

Comment:
We haven't set the value of `self.alert` yet. We should use `alert` here.

Reply:
Good catch ... I think I need a better Language Server integration :smile: